sunpy/map/sources/gong.py

```diff
@@ -40,7 +51,7 @@ class GONGSynopticMap(GenericMap):
     @classmethod
     def is_datasource_for(cls, data, header, **kwargs):
         return (str(header.get('TELESCOP', '')).endswith('GONG') and
-                str(header.get('CTYPE1', '').startswith('CRLN')))
+                str(header.get('CTYPE1', '')).startswith('CRLN'))
```

Comment:
Previously this was returning a string `'True'` and happened to work as there were not other GONG clients.